granulate_utils/linux/proc_events.py

```diff
@@ -116,7 +124,14 @@ def _listener_loop(self):
                 break
 
             for key, _ in events:
-                data = key.fileobj.recv(256)
+                try:
+                    # When stressed, reading from the socket can raise
+                    #   OSError: [Errno 105] No buffer space available
+                    # This seems to be safe to ignore, empirically no events were missed
+                    data = key.fileobj.recv(256)
+                except OSError as e:
+                    if e.errno == 105:
```

Comment:
Do you suppress other exceptions on purpose?

Reply:
woops